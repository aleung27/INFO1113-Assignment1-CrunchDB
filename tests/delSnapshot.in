SET a 8
SNAPSHOT
SET b 9
LIST ENTRIES
DEL b
LIST ENTRIES
CHECKOUT 1
LIST ENTRIES
DEL a
GET a
ROLLBACK 1
GET b
DEL a
LIST ENTRIES
