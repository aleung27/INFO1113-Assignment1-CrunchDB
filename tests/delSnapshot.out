> ok

> saved as snapshot 1

> ok

> b [9]
a [8]

> ok

> a [8]

> ok

> a [8]

> ok

> no such key

> ok

> no such key

> ok

> no entries

